interfaces/Portalicious/src/app/components/data-list/data-list.component.html

```diff
@@ -0,0 +1,47 @@
+<div class="mt-3 grid grid-cols-2 gap-x-5 border-t border-grey-300">
+  @for (item of data(); track $index) {
+    <p class="trim space-x-1 border-b border-grey-300 py-3">
+      @if (item.loading) {
+        <app-skeleton-inline />
+      } @else {
+        <strong> {{ item.label }}:</strong>
+        @if (item.tooltip) {
+          <app-info-tooltip
+            [message]="item.tooltip"
+            class="-ms-4"
```

Comment:
```suggestion
            class="ms-4"
```
?

Or is the prefixed `-` a working thing?